src/main/scala/is/hail/variant/VariantSampleMatrix.scala

```diff
@@ -65,18 +66,18 @@ object VariantSampleMatrix {
 
   def genGeneric(hc: HailContext): Gen[VariantSampleMatrix[Annotation]] =
     for (tSig <- Type.genArb.resize(3);
-      vsm <- VSMSubgen[Annotation](
```

Comment:
You should use `false` for `align_multiline_for` because that's what the `style_code.xml` says.